roles/edpm_neutron_ovn/tasks/bootstrap.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure the Openvswitch package is installed
+  ansible.builtin.package:
+    name: openvswitch
+    state: present
+
+- name: Ensure the OVS service is running
+  ansible.builtin.systemd:
+    name: openvswitch
+    state: started
```

Comment:
Shouldn't it be enabled to start at boot? Or it's only needed for bootstrap the role and then stopped?

Reply:
@rebtoor actually openvswitch service is enabled by edpm_bootstrap role. Here(like other network agents) the tasks were added in order to run the role in standalone. But yes the ovs service will keep running as we don't stop/disable it. And yes if needed we can add enabled: true here and in other network agents